packages/Pill/src/Pill.styles.js

```diff
@@ -0,0 +1,76 @@
+import { css } from "styled-components";
+import stylers from "@paprika/stylers";
+import tokens from "@paprika/tokens";
+
+const pillColorStyles = {
+  "#d7d7d7": css`
+    color: ${tokens.color.black};
+
+    svg {
+      fill: ${tokens.color.black};
```

Comment:
If we assume the `svg` is a Paprika `<Icon>`, then it will inherit the font color (above), so I don't think this line is needed.